public/components/Flyout/flyouts/createDetector.js

```diff
@@ -0,0 +1,833 @@
+    <EuiCallOut color={color}>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiIcon type={iconType} />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          {' '}
+          <EuiTextColor>{message}</EuiTextColor>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiCallOut>
+  );
+}
+
+function triggerCorrectCallOut(context) {
+  if (context.startedDetector) {
+    return informationCallOut(context, calloutType);
+  }
+  let calloutType = 'notValid';
+  if (
+    Object.keys(context.failures).length == 0 &&
+    Object.keys(context.suggestedChanges).length == 0
+  ) {
+    calloutType = 'valid';
+  }
+  for (let [key, value] of Object.entries(context.suggestedChanges)) {
+    if (key === 'detection_interval') {
+      let intervalMinutes = Math.ceil(value[0] / 60000) + 1;
+      if (isNaN(intervalMinutes) || intervalMinutes > MAX_INTERVAl_LENGTH_MINUTES) {
+        calloutType = 'maxInterval';
+        context.adConfigs.detection_interval = {
+          period: { interval: MAX_INTERVAl_LENGTH_MINUTES, unit: 'MINUTES' },
+        };
+      } else {
+        context.adConfigs.detection_interval = {
+          period: { interval: intervalMinutes, unit: 'MINUTES' },
+        };
+        if (
+          Object.keys(context.failures).length == 0 &&
+          Object.keys(context.suggestedChanges).length == 1
+        ) {
+          calloutType = 'valid';
+        }
+      }
+    }
+    if (
+      key === 'filter_query' &&
+      Object.keys(context.failures).length == 0 &&
+      Object.keys(context.suggestedChanges).length == 1
+    ) {
+      calloutType = 'filterQueryTooSparse';
+    }
+  }
+  return informationCallOut(context, calloutType);
+}
+
+function informationCallOut(context, callOut) {
+  const detectorID = context.detectorID;
+  if (context.startedDetector) {
+    return (
+      <EuiCallOut
+        title={'Anomaly Detector ' + context.adConfigs.name + ' has been created and started'}
+        size="m"
+        color="success"
+      >
+        <EuiFlexGroup>
+          <EuiFlexItem grow={false}>
+            <EuiIcon type="check" />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiTextColor color="subdued">
+              <span>
+                Anomaly detector has been created from the monitor and can be accessed{' '}
+                {
+                  <EuiLink
+                    style={{ textDecoration: 'underline' }}
+                    href={`${KIBANA_AD_PLUGIN}#/detectors/${detectorID}`}
+                    target="_blank"
+                  >
+                    {'here'} <EuiIcon size="s" type="popout" />
+                  </EuiLink>
+                }
+              </span>
+            </EuiTextColor>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiCallOut>
+    );
+  }
+  if (callOut === 'filterQueryTooSparse') {
+    return warningOrSubduedallOut(
+      'No Data is found with the current filter query used for the past 384 intervals, you' +
```

Comment:
what is the reason for 384 intervals? Seems this shouldn't be hardcoded into text here as well